### ISSUE TITLE
Conditional @mention in PRs

### ISSUE THREAD

**Issue by denelon** [microsoft/winget-pkgs#62962]
I'd like to have an option that I am not notified unless a reviewer @ me. Telling me that my pr needs to be approved is redundant since I manually allowed msftbot to make PRs on my behalf, so I know what will occur. A year or two ago, I would release multiple updates in a single day, so imagine the notifications I'd get then.

_Originally posted by @elibroftw in https://github.com/microsoft/winget-pkgs/issues/62956#issuecomment-1147981479_

**Comment by denelon** [comment]
I need to look and see if there is a good way to make the @mention configurable in bot messages.

Some users might need the notification while others may not.

**Comment by solomoncyj** [comment]
blocking the bot or unsubbing from prs?

**Comment by elibroftw** [comment]
I want to be auto-unsubbed from PRs and I don't want the bot mentioning me when things are going according to plan. There will always be a) manual approval required b) merging after approval. 

Developers should only be mentioned when things fail. Otherwise, a habit of deleting winget related notifications will end up with someone missing an important email about a failure.

**Comment by elibroftw** [comment]
I guess @vedantmgoyal2009's bot should also allow auto-unsubbing from the PR

**Comment by elibroftw** [comment]
Yes, I was talking about a fix from both you and the microsoft bot. If you unsub from from the start, and the bot never mentions me, then all is well.

**Comment by elibroftw** [comment]
The microsoft bot shouldn't need to @ anyways, since subscribed users would have gotten an email anyways.

**Comment by denelon** [comment]
I believe @Trenly has gone through the bot mentions and helped scrub the @ mentions unless there is a failure. The comments from the bots also contain a "Template/foo/bar" type of text in the body so e-mail rules can be built around specific comments.

Is that sufficient to consider this issue closed?

**Comment by elibroftw** [comment]
I guess so thanks.
